lib/model.js

```diff
@@ -976,7 +976,9 @@ const BookshelfModel = ModelBase.extend({
       attrs = key || {};
       options = _.clone(val) || {};
     } else {
-      (attrs = {})[key] = val;
+      attrs = {
+        [key]: val,
+      });
```

Comment:
You have an extra `)`. You should at least enable the linter in your IDE before submitting a PR, or run `npm run lint` as that would catch this small mistake.

Reply:
@ricardograca Sorry, I edited those lines on Github, so didn't find out the syntax issue.
I already fixed it and rebased the master.